components/lib/password/Password.js

```diff
@@ -262,8 +262,9 @@ export const Password = React.memo(React.forwardRef((props, ref) => {
                 <div className="p-password-meter">
                     <div className={`p-password-strength ${strength}`} style={{ width }}></div>
                 </div>
-                <div className="p-password-info">
-                    {infoTextState}
+
+                <div className={`p-password-strength-${strength}`}>
```

Comment:
No follow the pattern above it it should be `p-password-info ${strength}` with a space in between